client/src/main.jsx

```diff
@@ -29,13 +29,10 @@ import EditProfile from "./pages/EditProfile";
 import EditPersonalInfo from "./components/ProfileForm";
 import Admin from "./pages/Admin";
 import Score from "./pages/Score";
-import AdminStreetArtPage from "./pages/AdminStreetArtPage";
```

Comment:
Why do you delete this file import?
and the next.
Basically why delete all that is related to the street art page?

Reply:
Saw with Kevin, all good.